finger-exercises-react/react/tic-tac-toe/src/app/screens/PrevGames/index.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+
+import { matchesSuccess } from '../../../redux/PrevGames/actions';
+import MatchesService from '../../../services/MatchesService';
+
+import styles from './styles.scss';
+
+class PrevGames extends React.Component {
+  componentDidMount() {
+    MatchesService.getMatches().then(response => {
+      this.props.matchesSuccess(response.data);
+    });
+  }
+
+  winner(game) {
+    if (game.winner === 'tie') {
+      return 'Empate';
+    }
+    return game[game.winner];
+  }
+
+  render() {
+    const renderData = games =>
```

Comment:
`renderGames`?

Reply:
We generally avoid creating functions in the render method, you can move this to a new method.